### PR TITLE
first cut at adding ci-bots

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: recipes/cibots/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    version: 0.1
+
+source:
+    url: https://github.com/UV-CDAT/ci-bots/archive/0.1.tar.gz
+    sha256: c16d96619efa86fdabc23f49b6bff04a78e1d99a097a5fa8320504038dd6976b
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install
```
Comment (jakirkham) [inline recipes/cibots/meta.yaml]: Needs the arguments `--single-version-externally-managed --record record.txt` when using `setuptools`.

File: recipes/cibots/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    name: cibots
+    version: 0.1
+
+source:
+    url: https://github.com/UV-CDAT/ci-bots/archive/0.1.tar.gz
+    sha256: c16d96619efa86fdabc23f49b6bff04a78e1d99a097a5fa8320504038dd6976b
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (jakirkham) [inline recipes/cibots/meta.yaml]: Any reason not to build on Windows?

File: recipes/cibots/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - python
+    - requests
+
+test:
+  command:
+    - python run_tests.py -v2
+
+about:
+    home:  http://github.com/UV-CDAT/ci-bots
+    license: BSD
```
Comment (jakirkham) [inline recipes/cibots/meta.yaml]: Needs a version as well. Is it 2-Clause or 3-Clause?

Also the upstream repo doesn't seem to even note this is BSD in `setup.py`. 😟 Might want to take a look at that. Would also add a license file to the source repo and add it to the `MANIFEST.in` as well.

File: recipes/cibots/meta.yaml (hunk 2 above)
Comment (doutriaux1) [inline recipes/cibots/meta.yaml]: not really except non of my users or i have access to windows. I can turn it on if you want.

File: recipes/cibots/meta.yaml (hunk 1 above)
Comment (doutriaux1) [inline recipes/cibots/meta.yaml]: hum...
python setup.py install --single-version-externally-managed --record record.txt
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --single-version-externally-managed not recognized

Comment (doutriaux1) [inline recipes/cibots/meta.yaml]: I'm using distutils

**Comment by doutriaux1** [comment]
@jakirkham i think it's good to go now, please let me know.

File: recipes/cibots/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/cibots/meta.yaml]: Ok, then `setuptools` should be dropped from `build` then.

File: recipes/cibots/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{% set version = 0.2 %}
+{% set name = "cibots" %}
+package:
```
Comment (jakirkham) [inline recipes/cibots/meta.yaml]: nit: Please add a blank line between the Jinja and `package`.

**Comment by jakirkham** [comment]
Looks pretty good. Two minor comments. One about dropping `setuptools` from `build` and the other about formatting. Otherwise LGTM.

File: recipes/cibots/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    sha256: 93385842e6e5f127d70e1d60b4400d454fe0c81ace97bca4aa457ca7c9c648cc
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/cibots/meta.yaml]: Please drop `setuptools` if it is unused.

**Comment by jakirkham** [comment]
Looks pretty good. Two minor comments. One about dropping `setuptools` from `build` and the other about formatting. Otherwise LGTM.

**Comment by doutriaux1** [comment]
@jakirkham no more circle-ci integration?

**Comment by jakirkham** [comment]
Maybe try pushing again. Sometimes the webhooks fail.

**Comment by jakirkham** [comment]
Seems that CircleCI was enabled on your org's repo, @doutriaux1. Unfortunately CircleCI will only do builds for a branch/PR under one account not multiple unlike Travis CI and AppVeyor. Suspect that is part of the issue. Would recommend disabling CircleCI on your org's fork of staged-recipes. That way they can build under the conda-forge org's CircleCI account.

**Comment by jakirkham** [comment]
Merged anyways as the last build passed on CircleCI and the recent change is unlikely to have caused issues there.

**Comment by jakirkham** [comment]
Thanks @doutriaux1.

**Pull request merged**